templates/cisco_ios_show_ip_ospf_database_router.textfsm

```diff
@@ -0,0 +1,48 @@
+Value Filldown ROUTER_ID (\d+\.\d+\.\d+\.\d+)
+Value Filldown PROCESS_ID (\d+)
+Value Filldown AREA (\d+\.\d+\.\d+\.\d+|\d+)
+Value Filldown LSA_AGE (\d+)
+Value Filldown LSA_OPTIONS (.*)
+Value Filldown LSA_TYPE (.*)
+Value Filldown LSA_ID (\d+\.\d+\.\d+\.\d+)
+Value Filldown LSA_ADV_ROUTER (\d+\.\d+\.\d+\.\d+)
+Value Filldown LSA_SEQ_NUMBER (\S+)
+Value Filldown LSA_CHECKSUM (\S+)
+Value Filldown LSA_LENGTH (\d+)
+Value Filldown LSA_NUM_LINKS (\d+)
+Value LS_LINK_TYPE ((a Stub Network)|(a Transit Network)|(another Router \(point-to-point\)))
```

Comment:
You can probably just change this regex to `.+?` and then on the line that captures `LS_LINK_TYPE` end it with `$$`. Just simplifies the regex a bit. And with this new output, we should just capture everything rather than `Stub/Transit` since that doesn't work for all the available output

Reply:
Changes made via 87a1f37